CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 ## vNext (TBD)
 
 ### Deprecations
-* None
+* `Realm.Sync.reconnect(app)` has been deprecated and will be removed in the next major version. You can use `Realm.Sync.Session.reconnect()` instead.
```

Comment:
I believe it should be `Realm.App.Sync.Session.reconnect()` (missing the `App`).